src/main/frontend/client/components/BerriesIndex.js

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from "react"
 import { Link } from "react-router-dom"
 
-import BerriesTile from "./BerriesTile"
+import BerriesTile from "./BerryTile"
```

Comment:
For default imports this won't cause an error since we can name what we are importing anything we want, but it's a good idea to keep the name the same if we are able to.  In this case `import BerryTile from "./BerryTile"`